src/test_l1_dsFPD.c

```diff
@@ -486,22 +486,20 @@ void test_l1_dsFPD_positive_dsSetFPBlink (void)
 
     // Step 02: Retrieve the number of indicators from the profile file
     count = UT_KVP_PROFILE_GET_UINT8("dsFPD/Number_of_Indicators");
+    enableFPDIndicators(count);
 
     // Step 03: Set all valid indicators to dsFPD_STATE_ON using dsSetFPState()
     // Step 04: Call dsSetFPBlink() and loop through all valid indicators
-    // Step 05: Set all valid indicators to dsFPD_STATE_OFF using dsSetFPState()
     for (int i = 1; i <= count; i++)
     {
         snprintf(buffer, DS_FPD_KEY_SIZE, "dsFPD/SupportedFPDIndicators/%d/Indicator_Type", i);
         eIndicator =(dsFPDIndicator_t)UT_KVP_PROFILE_GET_UINT32(buffer);
         UT_LOG("\n In %s, Indicator: [%d]\n", __FUNCTION__, eIndicator);
-        result = dsSetFPState(eIndicator, dsFPD_STATE_ON);
```

Comment:
Please update the test procedures as per the changes.

Reply:
updated